_inc/client/writing/speed-up-site.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { FormFieldset } from 'components/forms';
+import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
+import { getModule } from 'state/modules';
+import { isModuleFound as _isModuleFound } from 'state/search';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { ModuleToggle } from 'components/module-toggle';
+
+const SpeedUpSite = moduleSettingsForm(
+	React.createClass( {
```

Comment:
We used `React.createClass` in the initial UI, but we should use ES6 classes for new components. For example
```js
export const SSO = moduleSettingsForm(
	class extends Component {
```
in `jetpack/_inc/client/security/sso.jsx`